README.md

```diff
@@ -27,8 +27,51 @@ Check out our [Contributing Guide] for how to get started.
     * [nothing yet...](#)
 
 # The List
-
-Nothing here yet...
+## Tooling ##
+
+### Register map over protocol code generator ###
+Often a MCU is connected to other devices (IC's) with protocols like I2C and SPI. These devices have their own register map. Register access for read and write is done together with providing the right address. Often these register maps are specified in datasheets (as PDF) and it's then left up to the programmar to implement everything in code. Sometimes C files are provided as well.
+
+This proposal is for a tool that can generate code for these register accesses. Ideally this should work for multiple protocols. Therefor it would be good to consider design for I2C and SPI (one of the two most common protocols) with each step of the implementation.
```

Comment:
```suggestion
This proposal is for a tool that can generate code for these register accesses. Ideally this tool should work for multiple protocols. Therefore it would be good to consider design for I2C and SPI (two of the most common protocols) with each step of the implementation.
```